server/database/models/User.js

```diff
@@ -0,0 +1,35 @@
+const mongoose = require('mongoose');
+const bcrypt = require('bcrypt');
+
+const UserSchema = new mongoose.SchemaType({
+  email: {
+    type: String,
+    default: '',
+  },
+  password: {
+    type: String,
+    default: '',
+  },
+  name: {
+    type: String,
+    default: '',
+  },
+  isDeleted: {
```

Comment:
what is deleted for ?

Reply:
When a user deletes their account, this will change to true. We may not need it after alll, but I will leave it in for now :)